### PR TITLE
fix: Typography copyable in clipboard incorrect

### PR DESCRIPTION
**Pull request by opopeieie**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md)]

### 🤔 This is a ...

- [ ] New feature
- [x] Bug fix
- [ ] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [x] Test Case
- [ ] Branch merge
- [ ] Other (about what?)

### 💡 Background and solution
https://codesandbox.io/s/sleepy-dawn-qedrs?file=/src/App.js

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |    Fix Typography `copyable` incorrect copy text after children is updated.      |
| 🇨🇳 Chinese |  修复  Typography `copyable` 时 `children` 内容变化后复制内容没变的问题。      |

### ☑️ Self Check before Merge

⚠️ Please check all items below before review. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

### CODE REVIEW

**Comment by afc163** [comment]
Could you add a test case?

File: components/typography/Base/index.tsx [hunk 1]
```diff
@@ -196,10 +196,7 @@ const Base = React.forwardRef((props: InternalBlockProps, ref: any) => {
     e?.preventDefault();
     e?.stopPropagation();
 
-    if (copyConfig.text === undefined) {
-      copyConfig.text = String(children);
-    }
-    copy(copyConfig.text || '');
+    copy(copyConfig.text || String(children) || '');
```
Comment (afc163) [inline components/typography/Base/index.tsx:199]: copyConfig.text 为 null 和 false 和 `''` 时不应该有复制文本，只有 undefined 时才会到 children

Comment (opopeieie) [reply components/typography/Base/index.tsx:199]: 如果copyConfig.text需要校验的话，children是不是也同样要判断null false true undefined number这些类型，所以还不如让开发者自己控制就好，限制太死这里逻辑就多了吧

Comment (afc163) [reply components/typography/Base/index.tsx:199]: 不过复制文本为空时，其实这时候应该就不出复制图标了。感觉单独设置复制文本为空确实没什么意义。

Comment (opopeieie) [reply components/typography/Base/index.tsx:199]: 也是。而且目前这个copy功能不能复制空字符串可太秀了，空字符串粘贴出来的是上次复制内容。。。看你们怎么考虑吧

**Pull request merged**